tests/ImportsNotSeparated/imports-with-directives.ts

```diff
@@ -0,0 +1,6 @@
+'use strict';
```

Comment:
I guess having two directives will never exactly happen, I tested locally and worked with just one too.
If there's a comment in front of it, the comment will end up being moved to a new line above the directive, for RSC most probably this will be incompatible, so no comments explaining why "use client"; 😅

Reply:
Looking good. Well done!
<img width="1178" alt="image" src="https://user-images.githubusercontent.com/411161/201544047-896a95cc-2db4-43d5-ab53-45fa675c61e7.png">